### PR TITLE
Updated OAuth docs regarding selection of application type

### PR DESCRIPTION
**Pull request by Rohit-RA-2020**
## What does this PR do?

Updates the doc regarding selection of application type in OAuth providers

## Test Plan

Doc updates

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by Rohit-RA-2020** [comment]
Hey @gewenyu99, you can review this

File: app/views/docs/authentication.phtml [hunk 1]
```diff
@@ -307,6 +307,11 @@ try await account.createOAuth2Session(provider: "amazon")</code></pre>
     If there is already an active anonymous session, the new session will be attached to it. If there are no active sessions, the server will attempt to look for an account with the same email address as the email received from the OAuth2 provider and attach the new session to the existing account. If no matching account is found - the server will create a new account.
 </p>
 
+<div class="notice">
+    <h3>Note</h3>
+    <p>While you setup OAuth2 providers, you need to enter App ID and App Secret in your Appwrite console. You need to choose a web application in your respective 3rd party OAuth2 provider (Ex: Google) irrespective of the target platform you are building for. This is beacuse the communication is between Appwrite (Web App) and the OAuth2 provider</p>
```
Comment (gewenyu99) [inline app/views/docs/authentication.phtml:312]:
```suggestion
    <p>You should select <b>web application</b> as the application type in your OAuth2 provider, even if you're building a mobile or flutter application. The OAuth2 provider will communicate with Appwrite directly, which is a web application.</p>
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

**Review by gewenyu99 — APPROVED** [review]

**Review by gewenyu99 — APPROVED** [review]

File: app/views/docs/authentication.phtml [hunk 2]
```diff
@@ -307,6 +307,11 @@ try await account.createOAuth2Session(provider: "amazon")</code></pre>
     If there is already an active anonymous session, the new session will be attached to it. If there are no active sessions, the server will attempt to look for an account with the same email address as the email received from the OAuth2 provider and attach the new session to the existing account. If no matching account is found - the server will create a new account.
 </p>
 
+<div class="notice">
+    <h3>Note</h3>
```
Comment (gewenyu99) [inline app/views/docs/authentication.phtml:311]: On second thought, can you suggest some better titles for this than "Note"? We don't use generic titles like this.

**Review by gewenyu99 — APPROVED** [review]

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
See previous comment

**Pull request closed without merging**